src/ProgressBar.js

```diff
@@ -116,15 +120,24 @@ const ProgressBar = React.createClass({
         {this.props.label}
       </InterpolateClass>
     );
-  },
-
-  renderScreenReaderOnlyLabel(label) {
-    return (
-      <span className="sr-only">
-        {label}
-      </span>
-    );
   }
 });
 
+/**
+ * Custom propTypes checker
+ */
+function onlyProgressBar(props, propName, componentName) {
+  if (props[propName]) {
+    let error;
+
+    React.Children.forEach(props[propName], (child) => {
+      if (child.type.displayName !== ProgressBar.displayName) {
+        error = new Error(`Children of ${componentName} can contain only ProgressBar components`);
```

Comment:
Is there a way to have an identifier, whether display name or a type or something?